modules/apps/object/object-api/src/main/java/com/liferay/object/action/executor/ObjectActionExecutor.java

```diff
@@ -28,6 +31,14 @@ public void execute(
 			JSONObject payloadJSONObject, long userId)
 		throws Exception;
 
+	public default Set<String> getAllowedObjectDefinitionNames() {
+		return Collections.emptySet();
+	}
```

Comment:
@murilostodolni rename to `getObjectDefinitionNames` because we don't have conflicts and ensure the default return maches the empty one retrieved by Client Extensions.